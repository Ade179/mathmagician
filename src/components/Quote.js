import React, { PureComponent } from 'react';
import './CSSmodules/quote.css';

class Quote extends PureComponent {
  render() {
    return (
      <>
        <section className="sect1">
          <div className="quotes">
            <h2 className="mathematician">Mathematics is not about numbers, equations, computations or algorithms; it is about understanding - William Paul Thurston.</h2>
          </div>
        </section>
      </>
    );
  }
}

export default Quote;
