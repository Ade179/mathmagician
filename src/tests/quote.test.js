import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('should match the snapshot', () => {
  const quote = renderer.create(<Quote />);
  const tree = quote.toJSON();
  expect(tree).toMatchSnapshot();
});
