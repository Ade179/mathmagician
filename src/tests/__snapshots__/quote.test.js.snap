// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<section
  className="sect1"
>
  <div
    className="quotes"
  >
    <h2
      className="mathematician"
    >
      Mathematics is not about numbers, equations, computations or algorithms; it is about understanding - William Paul Thurston.
    </h2>
  </div>
</section>
`;
